exareme2/algorithms/exaflow/exaflow_registry.py

```diff
@@ -0,0 +1,26 @@
+from exareme2.utils import Singleton
+
+
+class ExaflowRegistry(metaclass=Singleton):
+    def __init__(self):
+        self._registry = {}
+
+    def register(self, func):
+        key = f"{func.__module__.split('.')[-1]}_{func.__name__}"
```

Comment:
!! 💯 nice!

Could you please also write a small comment on what this creates with an example?